### PR TITLE
docs: Update vGPU use-case

### PR DESCRIPTION
**Pull request by zvonkok**
Now that #4213 is merged we need updated documentation for vGPU time-sliced or vGPU MIG-backed.

Fixes: #4343

Signed-off-by: Zvonko Kaiser <zkaiser@nvidia.com>

### CODE REVIEW

**Comment by zvonkok** [comment]
@liubin @Jimmy-Xu @bergwolf PTAL

**Comment by liubin** [comment]
@zvonkok :

> time="2022-05-31T06:57:11.163444763Z" level=fatal msg="file=\"docs/use-cases/NVIDIA-GPU-passthrough-and-Kata.md\": failed to find heading for link \"list-all-available-vGPU-instances\" ({Doc:docs/use-cases/NVIDIA-GPU-passthrough-and-Kata.md Address:list-all-available-vGPU-instances ResolvedPath: Description:snippet Type:internal-link})" commit=f705289b69a8e5b69e09454771c1c74fbced92da name=kata-check-markdown pid=33677 source=check-markdown version=0.0.1

All links to `list-all-available-vGPU-instances` should be `list-all-available-vgpu-instances`.

File: docs/use-cases/NVIDIA-GPU-passthrough-and-Kata.md [hunk 1]
```diff
@@ -246,9 +246,230 @@ Use the following steps to pass an NVIDIA GPU device in pass-through mode with K
...
++--------------+-----------+----------------+------------+
+| 0000:41:00.4 |         1 | GRID A100D-40C | nvidia-697 |
+
+| BDF          | INSTANCES | NAME           | DIR        |
++--------------+-----------+----------------+------------+
+| 0000:41:00.4 |         1 | GRID A100D-80C | nvidia-698 |
+
+```
+
+Change to the mdev_supported_types directory for the virtual function on which
```
Comment (liubin) [inline docs/use-cases/NVIDIA-GPU-passthrough-and-Kata.md:347]: mdev_supported_types -> `mdev_supported_types`, or the markdown checker may complains typo.

File: docs/use-cases/NVIDIA-GPU-passthrough-and-Kata.md [hunk 2]
```diff
@@ -246,9 +246,230 @@ Use the following steps to pass an NVIDIA GPU device in pass-through mode with K
...
+you want to create the vGPU on. Takin the first output as an example:
+
+```sh
+$ cd virtfn0/mdev_supported_types/nvidia-692
+$ UUID=$(uuidgen)
+$ sudo bash -c "echo $UID > create"
```
[the remark this answers is not in the document]
Comment (zvonkok) [reply docs/use-cases/NVIDIA-GPU-passthrough-and-Kata.md:353]: Of course UUID not the current user id ... Thanks good catch!

File: docs/use-cases/NVIDIA-GPU-passthrough-and-Kata.md [hunk 3]
```diff
@@ -246,9 +246,230 @@ Use the following steps to pass an NVIDIA GPU device in pass-through mode with K
...
+
+                if [ $AVAIL -gt 0 ]; then
+                        echo "| BDF          | INSTANCES | NAME           | DIR        |"
+                        echo "+--------------+-----------+----------------+------------+"
+                        printf "| %12s |%10d |%15s | %10s |\n\n" "$BDF" "$AVAIL" "$NAME" "$DIR"
+                fi
+
+        done
+
+        read
```
[the remark this answers is not in the document]
Comment (zvonkok) [reply docs/use-cases/NVIDIA-GPU-passthrough-and-Kata.md:308]: Removed, if needed users can `| less` or `| more` which makes more sense than me creating something artificial.

**Comment by zvonkok** [comment]
@fidencio @liubin /test :)

**Comment by liubin** [comment]
@zvonkok 

Please again:

```
[kata-spell-check.sh:275] WARNING: Word 'Takin': did you mean one of the following?: Akin, T akin, Ta kin, Ta-kin, Taking, Taken, Akita
[kata-spell-check.sh:275] WARNING: Word 'avilable': did you mean one of the following?: available, violable, viable
[kata-spell-check.sh:275] WARNING: Word 'guarantedd': did you mean one of the following?: guaranteed, guarantee
[kata-spell-check.sh:275] WARNING: Word 'isnstances': did you mean one of the following?: instances, assistances, resistances, distances, instance
[kata-spell-check.sh:275] WARNING: Word 'isntance': did you mean one of the following?: instance, assistance, distance, resistance, antiscience
[kata-spell-check.sh:275] WARNING: Word 'ouput': did you mean one of the following?: output, putout, out
[kata-spell-check.sh:275] WARNING: Word 'ouput': did you mean one of the following?: output, putout, out
[kata-spell-check.sh:275] WARNING: Word 'sysfs': did you mean one of the following?: sysops
[kata-spell-check.sh:275] WARNING: Word 'vGPUs': did you mean one of the following?: vagus
[kata-spell-check.sh:275] WARNING: Word 'vGPUs': did you mean one of the following?: vagus
```

**Comment by zvonkok** [comment]
@liubin Sorry in the end it is an error. All good.

**Comment by zvonkok** [comment]
```
+ die 'packages not installed'
+ local 'msg=packages not installed'
+ echo 'ERROR: packages not installed'
ERROR: packages not installed
+ exit 1
```
@BbolroC  PTAL s390x failing, this PR is just a documentation *.md update.

**Comment by BbolroC** [comment]
@zvonkok Hi, thanks for the message. Please ignore the result. I will have a look at it later.

**Comment by BbolroC** [comment]
@zvonkok fixed. thanks~!

**Comment by zvonkok** [comment]
@jodh-intel @liubin @Jimmy-Xu @bergwolf All tests green, PTAL

**Review by GabyCT — APPROVED** [review]

**Review by liubin — APPROVED** [review]

**Pull request merged**